ingestion_server/ingestion_server/elasticsearch_models.py

```diff
@@ -280,6 +269,19 @@ def get_aspect_ratio(height, width):
             aspect_ratio = Image.AspectRatios.SQUARE.name
         return aspect_ratio.lower()
 
+    @staticmethod
+    def get_extension(url):
+        """
+        Get the extension from the last segment of the URL separated by a dot.
+        TODO: Use the `filetype` field once the following issue is completed:
+        https://github.com/WordPress/openverse-catalog/issues/536
+        """
+        extension = url.split(".")[-1].lower()
+        if "/" in extension or extension is None:
```

Comment:
For null safety we need the None check to go first:

```
>>> '/' in None
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: argument of type 'NoneType' is not iterable
```

```suggestion
        if not extension or "/" in extension:
```